site/docs/v1/tech/lambdas/index.adoc

```diff
@@ -111,9 +204,25 @@ function populate(jwt, user, registration) {
   console.info('jwt:' + object2String(jwt));
 ----
 
+=== HTTP Request Limitations
+
+When making HTTP requests, if you make an API call in a lambda to an endpoint that triggers the same lambda, it will call itself recursively. This does not end well.
+
+HTTP requests will time out after two seconds.
+
+The `fetch` method in a lambda does not implement the https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API[entire `fetch` API] as implemented in a browser.
+The first argument to `fetch` must always be a string URL.
+Only the following options are supported:
+
+* `method`, which defaults to `GET`
+* `headers`, which defaults to null
+* `body`, which must be a string
```

Comment:
I don't know that we want to describe the issue like this, I don't want to describe how to DoS the system. Also, we have protections in place for this, so it should fail. 

Could we just mention that you can call FusionAuth API over localhost within a Lambda is allowed, but do not call the same API that is currently invoking you, because it will fail.

Reply:
Sure, will make that change.